fearless/Common/Services/EraCountdownService/EraCountdownService.swift

```diff
@@ -0,0 +1,203 @@
+import RobinHood
+import FearlessUtils
+import SoraKeystore
+
+protocol EraCountdownServiceProtocol {
+    func fetchCountdownOperationWrapper() -> CompoundOperationWrapper<UInt64>
+}
+
+enum EraCountdownServiceError: Error {
+    case noData
+}
+
+final class EraCountdownService: EraCountdownServiceProtocol {
```

Comment:
Let't rename this class to EraCountdownOperationFactory for consistency because it just creates operations and doesn't perform any other logic